docs/api_docs/user.md

```diff
@@ -30,60 +35,44 @@
 #### `GET user/logout`
 - Logs user out of Meetify
 - Ends user session
+- Status Codes:
+    - `204` - Successful logout
 
 #### `user/{id}/profile`
 `GET`
 - Retrieves user profile data for the requested id
 - Does not require authentication
+- Status Codes:
+    - `200` - Successfully returned user
+    - `404` - Requested user not found
 
 `POST`
 - Updates user profile data for the requested id
 - Requires the requested user to be logged in
+ - Status Codes:
+    - `200` - Successfully updated user
+    - `401` - Requested user is not logged in
 - Sample `POST` body:
 ```
 {
     "ZipCode": 45219
 }
 ```
-
-#### `GET user/update-liked-songs`
-- Updates the Meetify Database with any changes to the current user's liked songs.
+*Use of any verbs other than GET or POST will result in* `405` - Invalid request method
 
 ### Spotify-related user endpoints
 #### `GET user/link-account`
 - Links Spotify account with Meetify user that is currently logged in
 - Redirects to `/callback`
+- Status Codes:
+    - `200` - Successfully linked Spotify account
 
 #### `GET user/refresh-token`
```

Comment:
I don't really understand this endpoint; when would I use this / why would I need to refresh this?

Reply:
I primarily made this endpoint for development and testing purposes. It likely won't be used in the front-end.